lab-ron/lib/greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// function cmd() {
+  // const name = process.argv[2] || 'Ron Oh';
+  // console.log(name);
+  // return 'hello ' + name;
+// }
+
```

Comment:
Our assignment submissions are simulating real-world code submissions - Please go through your code and remove anything that is not related to the assignment before submission.